data/json/items/tools.json

```diff
@@ -482,7 +482,7 @@
     "material": [ "steel", "wood" ],
     "symbol": "/",
     "color": "light_gray",
-    "qualities": [ [ "AXE", 2 ] ],
+    "qualities": [ [ "AXE", 1 ] ],
```

Comment:
You think that a specialized, sharp weapon is on par with the crude makeshift tools with uneven and not particulary sharp cutting edge in terms of woodcutting? The same for inferior version.

Reply:
There is a reason that battle axes are not used for wood cutting. The blade is general much lighter and thinner than on a splitting axe, and even on most hatchets. Either way I at least went for consistency. Note that "battle axe is equal to stone axe" was how it as to begin with.